### DISCUSSION TITLE
Is there any way to disable the back to previous page animation on ios devices

### DISCUSSION THREAD

**Discussion by Sinevik** [remix-run/react-router#11826] [Proposals]
Is there any way to disable the back to previous page animation on ios devices? An analogy is to click on the back arrow in the browser. On ios devices, there is a backwards page flip animation. I don't like it very much. Also, it's a little bit laggy

 "react-router-dom": "^6.6.2",

**Comment by sergiodxa** [comment]
React Router has no animation, that's done by the browser, AFAIK there's no way to control that from within the page, is up to the user to disable the browser animations (if it's even possible)
